### DISCUSSION TITLE
Share your ideas or thoughts

### DISCUSSION THREAD

**Discussion by cedricve** [kerberos-io/agent#56] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CAI79** [comment]
Hi! How can I contribute to the project? I have translated it into Russian, but I do not have the rights to commit language file.

**Reply by cedricve** [reply, marked as answer]
Russian has been added.

**Comment by dridri** [comment]
Hi, are there any plans to make a self-hosted lightweight low-cost live viewer ?

Just installed two Agents at home and was looking for a self-hosted UI to watch both livestreams on a single page, it seems that the only way to do it is to buy the complete Hub license which is a little bit expensive for all the other features that I'll never use.

edit : well, I could just use any random rtsp viewer actually

**Reply by cedricve** [reply]
Indeed, we'll be adding a basic view layer on the Kerberos Vault solution (which might fulfil your need), but indeed majority of functionalities will go in our paying product.

**Reply by Remmom** [reply]
Hi, I am new to Kerberos but have managed to install it and add some cameras. Your documentation seems to be limited to introducing the products you are offering and the installation process. I want to know if there is documentation on how to integrate Kerberos with 3rd party plugins and how to implement other services like motion detection?

**Reply by cedricve** [reply]
> Hi, are there any plans to make a self-hosted lightweight low-cost live viewer ?
> 
> 
> 
> Just installed two Agents at home and was looking for a self-hosted UI to watch both livestreams on a single page, it seems that the only way to do it is to buy the complete Hub license which is a little bit expensive for all the other features that I'll never use.
> 
> 
> 
> edit : well, I could just use any random rtsp viewer actually 

You can also substibe to our saas version which starts at 2$/month. Forgot to mention this

**Reply by Remmom** [reply]
I am using Kerberos factory. I like the system but I would really appreciate if I can get more help through the documentation or videos. I am going to install the Hub and Vault too. I haven't checked out the SaaS version. What is included in it?

**Reply by cedricve** [reply]
Kerberos Hub has now a trial license which avoids any licenses fees for deployments smaller then 8 cameras. We are making a cool tutorial to get it up and running on microk8s with a single command. Work can be found here: https://github.com/kerberos-io/deployment

**Comment by noxare** [comment]
Hello, i can't find an option to blackening unwanted areas. 
In Germany its not allowed to record some areas and they has to disabled with black bars.

**Reply by cedricve** [reply]
Thanks great feature suggestion! we'll add this to our roadmap!

**Reply by cedricve** [reply]
We will be adding an option for face blurring.

**Comment by alex-vald** [comment]
Hello! I think kerberos.io is a wonderful product. But as I understood according to editions page, there is a huge gap in plans and prices.
We would be almost satisfied with one for 49  Eur, but with self-hosted option and expanded time. And looks like there is nothing in between with the next one for 999 Eur.\month. 49 is nothing and 999/mo is huge price for us!
Is it possible to have something like self-hosted all components (including hub, but excluding factory, we don`t need it), 180 days of storage at self-hosted Vault, unlimited gb\day of uploads and 100 Eur/mo for example?

**Comment by kiosvantra** [comment]
hi @cedricve. I was wondering if it possible to have another alternatives to minio, for local storage. kind regards,

**Comment by conexto** [comment]
Hello
I hope to contirbute to develop this project
